lib/engine/game/g_18_oe/step/buy_sell_par_shares.rb

```diff
@@ -1,38 +1,102 @@
 # frozen_string_literal: true
 
 require_relative '../../../step/buy_sell_par_shares'
+require_relative '../../../action/convert'
 
 module Engine
   module Game
     module G18OE
       module Step
         class BuySellParShares < Engine::Step::BuySellParShares
           def actions(entity)
+            return corporation_actions(entity) if entity.corporation?
             return [] unless entity == current_entity
+            return [] if @round.current_actions.any? { |x| x.instance_of?(Engine::Action::Par) }
```

Comment:
Shouldn't the code call pass! after process_par_action, rather than going back here?

Reply:
Yeah that would be a better way of doing that